created a docs folder to add wireframes to along with a wireframe for…

Rhardin378 opened the pull request:
… SearchResults

## Related Issue
- Closes: #34 

## Description
I created the wireframe for the search results page.  I posted it on the brainstorming FigJam board.  I created a Readme.md file with the help of AI to provide a more detailed explanation. (If the team wants we can delete this folder after closing this ticket.   I placed a picture of the wireframe in a docs folder on the root, in order to make a PR to close the design ticket.
## Screenshots/Videos
<img width="370" height="747" alt="SearchResultsWireFrame" src="https://github.com/user-attachments/assets/7ea85338-80d6-4c86-ab2f-fe47ed51cbbd" />

[Figjam Board] (https://www.figma.com/board/LBXggp7T2UD5rUonHWUEsQ/Brainstorm-FigJam?node-id=0-1&p=f&t=2ASTzYarf1qh3bhT-0)

## How to Test
- List out steps to reproduce or test these changes: 
1. Git pull the latest changes
2. Navigate to `/docs/wireframes/SearchResults/`
3. Review the README.md file for complete wireframe documentation.

bethanyann commented:
A PR for the wireframe wasn't necessary so unless you think it's important to have a README file for it, this PR can be closed and you can proceed with starting to write code for the page itself. 

If you think the README is useful then we can keep this PR, but uploading the image of the wireframe can be deleted as that's not super useful to have in the rep;ository.

bethanyann reviewed (changes requested):
Let me know what you think about my comment!

Rhardin378 commented:
> A PR for the wireframe wasn't necessary so unless you think it's important to have a README file for it, this PR can be closed and you can proceed with starting to write code for the page itself.
> 
> If you think the README is useful then we can keep this PR, but uploading the image of the wireframe can be deleted as that's not super useful to have in the repository.

I think it is fairly self-explanatory, given the arrows and flow diagram.  I just wanted to close the ticket.  We can delete the PR and not merge the docs folder.  I added the screenshot to a comment on the original ticket and moved it to complete.  Thanks!
